README.md

```diff
@@ -1,55 +1,56 @@
 # Autoware (Architecture Proposal)
 
-meta-repository for Autoware architecture proposal version
-
 ![autoware](https://user-images.githubusercontent.com/8327598/69472442-cca50b00-0ded-11ea-9da0-9e2302aa1061.png)
 
-# What's this
-
-This is the source code of the feasibility study for Autoware architecture proposal.
-
-> **WARNING**: This source is solely for demonstrating an architecture proposal. It should not be used to drive cars. 
+A meta-repository for the Autoware architecture proposal feasibility study created by Tier IV. For more details about the architecture itself, please read this [overview](/design/Overview.md).
 
-> **NOTE**: The features in [AutowareArchitectureProposal.iv](https://github.com/tier4/AutowareArchitectureProposal.iv) will be merged into Autoware.Auto.
+> **WARNING**: All source code relating to this meta-repository is solely intended to demonstrate a potential new architecture for Autoware, and should not be used to autonomously drive a real car!
+> 
+> **NOTE**: The features within the [AutowareArchitectureProposal.iv repository](https://github.com/tier4/AutowareArchitectureProposal.iv) are planned to be merged into [Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto).
 
-Architecture overview is [here](/design/Overview.md).
-
-# How to setup
+# Installation Guide
 
 ## Requirements
 
 ### Hardware
 
-- x86 CPU (8 or more cores)
-- 16 GB or more of memory
-- Nvidia GPU (4GB or more of memory) \*optional
-  - When you use following packages, GPU is mandatory:
-    - lidar_apollo_instance_segmentation
-    - traffic_light_ssd_fine_detector
-    - cnn_classifier
+- x86 CPU (8 cores)
+- 16GB RAM
+- Nvidia GPU (4GB RAM) \*optional
+  - Although not required to run basic functionality, a GPU is mandatory in order to run the following components:
+    - lidar_apollo_instance_segmentation
+    - traffic_light_ssd_fine_detector
+    - cnn_classifier
 
-If cuda or tensorRT is already installed, it is recommended to remove it.
+> Note that the specifications above are the minimum requirements. Performance will be improved with more cores, RAM and a higher-spec graphics card.

```

Comment:
Maybe we can just change the header "Requirements" to "Minimum Requirements". Leaving just the note regarding, improved performance.